modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -0,0 +1,128 @@
+= Migrating User Key-based Encryption to Master Key Encryption
+
+== Why Should I Move Away From User Key-based Encryption?
+
+NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. While it is a bit more secure than a central encryption approach, User key-based encryption has some disadvantages. It blocks some additional functions such as the integration of an online editor like LibreOffice or OnlyOffice into ownCloud and can cause problems when sharing files with groups. Therefore Master-key-based encryption is now the recommended setup fr all new insallations.
```

Comment:
There's a spelling mistake at the end of the last sentence in this paragraph; "setup fr all new insallations" should read as "setup for all new insallations".

Reply:
done